### PR TITLE
feat(http): Add utility to map http responses to typed exceptions

### PR DESCRIPTION
**Pull request by wdullaer**
Let me know if you agree with the mappings, or if you'd like to add more to the default mappings.

### CODE REVIEW

File: modules/ex-http/src/clj/exoscale/ex/http.clj [hunk 1]
```diff
@@ -0,0 +1,60 @@
+(ns exoscale.ex.manifold
```
Comment (bcachet) [inline modules/ex-http/src/clj/exoscale/ex/http.clj:1]: namespace should be exoscale.ex.http

Comment (wdullaer) [reply modules/ex-http/src/clj/exoscale/ex/http.clj]: yeah :-)

**Comment by wdullaer** [comment]
Will do

**Comment by mpenet** [comment]
about the mappings, you took them from blueprint? If not it would be good to ensure they match somewhat.

**Comment by wdullaer** [comment]
I did the mappings by hand.
I cross referenced them with the table here: https://github.com/exoscale/blueprint/blob/17a2647f5f36e6629f2354d0ff6b662b81f546c0/doc/error.md#errors-in-command-handlers (I made two changes compared to the version you reviewed)

**Pull request merged**